src/Argument/ClassNameInterface.php

```diff
@@ -9,5 +9,5 @@ interface ClassNameInterface
      *
      * @return string
      */
-    public function getValue() : string;
+    public function getClassName() : string;
```

Comment:
took the liberty to change this method name, it's an internal class so it shouldn't be a problem.

Reply:
I may need to change this back as even though it's mainly internal, it's public api and open for use, I'll do that when I merge though.

4.x removes the class completely for a better raw argument system

Reply:
> I'll do that when I merge though.

👍